### DISCUSSION TITLE
Save the results of a comparison in native WinMerge format?

### DISCUSSION THREAD

**Discussion by njminchin** [WinMerge/winmerge#1167] [Q&A]
Is it possible to save the current comparison results as a native WinMerge file so that I can open it again and keep intact everything as it is? I've hidden a bunch of results as I've already fixed them manually and I want to maintain this. (If I refresh the compare it will unhide these files which I don't want- my files masters aren't housed locally and I'm editing them manually remotely after checking the differences)

**Comment by sdottaka** [comment]
Unfortunately, WinMerge does not have the ability to save and restore comparison results, hidden states, etc.

I want that feature too, but it won't be implemented anytime soon unless someone makes a Pull Request.

**Comment by spowers225** [comment]
This would be a great feature.
